utilities/bigquery.py

```diff
@@ -16,6 +18,27 @@ def get_likes(id_token):
         where
             json_value(data, '$.uid') = '{uid}'
     '''
-    results = db.query(query)
+    results = db.query(query, job_config=job_config)
     likes = [json.loads(row[0]) for row in results]
     return likes
+
+def get_song(id_token, time_of_day, likes):
+    """Get song recommendation for the logged in user."""
+    uid = auth.verify_id_token(id_token)['uid'] # pylint: disable=unused-variable
+    like_ids = ','.join(
+        [str(like['id']) for like in likes if like['time_of_day'] == time_of_day]
+    )
+    likes_string = f'\nand id not in ({like_ids})' if like_ids else ''
+    query = f'''
+        select
+            *
+        from
+            `faf-starter.noon.recommendations`
+        where
+            time_of_day = '{time_of_day}'{likes_string}
+        limit
+            25
+    '''
+    results = db.query(query, job_config=job_config)
+    song = {f[0]: f[1] for f in random.choice(list(results)).items()}
```

Comment:
use descriptive variable names to make this code more readable - it's hard to quickly see what's going on in this line